src/Host/Client/Impl/Microsoft.R.Host.Client.csproj

```diff
@@ -198,6 +193,10 @@
       <Project>{8d408909-459f-4853-a36c-745118f99869}</Project>
       <Name>Microsoft.Common.Core</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\R\Core\Impl\Microsoft.R.Core.csproj">
+      <Project>{0c4bce1d-3cb8-4e2a-9252-58784d7f26a5}</Project>
+      <Name>Microsoft.R.Core</Name>
```

Comment:
After I am done with remote intellisense (which should be today) I will take all Host.Client related code from R.Support to R.Editor and R.Support won't be needed and can be reused.
Or we can rename R.Core -> R.Language.Core and reuse R.Core name.